examples/avmnist_multimodal/model.py

```diff
@@ -0,0 +1,52 @@
+"""
+Define the Multimodal Deep Learning (MMDL) model and configure model hyperparameters.
+References: 1. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_simple_late_fusion.py
+            2. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_low_rank_tensor.py
+            3. https://github.com/pliang279/MultiBench/blob/main/examples/multimedia/avmnist_multi_interac_matrix.py
+"""
+
+from kale.embed.feature_fusion import Concat, LowRankTensorFusion, MultiplicativeInteractions2Modal
+from kale.embed.image_cnn import LeNet
+from kale.pipeline.base_nn_trainer import MultiModalTrainer
+from kale.pipeline.multimodal_deep_learning import MultiModalDeepLearning
+from kale.predict.decode import MLPDecoder
+
+
+def get_model(cfg, device):
+    """
+    Builds and returns an MMDL model.
+
+    Args:
+        cfg: A YACS config object.
+    """
+    encoders = [
+        LeNet(cfg.MODEL.LENET_IN_CHANNELS, cfg.MODEL.CHANNELS, cfg.MODEL.LENET_ADD_LAYERS_IMG),
+        LeNet(cfg.MODEL.LENET_IN_CHANNELS, cfg.MODEL.CHANNELS, cfg.MODEL.LENET_ADD_LAYERS_AUD),
+    ]
+
+    if cfg.MODEL.FUSION == "late":
+        fusion = Concat()
+        head = MLPDecoder(
+            cfg.MODEL.MLP_IN_DIM, cfg.MODEL.MLP_HIDDEN_DIM, cfg.MODEL.OUT_DIM, include_additional_layers=False
+        )
+    elif cfg.MODEL.FUSION == "tensor_matrix":
```

Comment:
syn the name here, discuss with me if needed

Reply:
fixed